olm/build-image.sh

```diff
@@ -9,6 +9,10 @@ version="{{{ .major }}}.{{{ .minor }}}.{{{ .patch }}}"
 registry="container-registry.oracle.com/olcne"
 docker_tag=${registry}/${name}:v${version}
 
+if [ -f "/etc/yum.repos.internal.d/ol_artifacts.repo" ]; then
+    cp /etc/yum.repos.internal.d/ol_artifacts.repo ./
+fi
+
```

Comment:
Use the following in the docker build command instead 

		--volume /etc/yum.repos.internal.d:/etc/yum.repos.internal.d \
		--volume /etc/yum.conf:/etc/yum.conf \

Reply:
I am intentionally not using the way you suggested bcoz it becomes difficult to build locally whether it's us or an end-user outside of oracle. Your suggested way leads to an error when building image outside of CI/CD:
```
++ podman build --pull --build-arg https_proxy= --build-arg version=0.4.0 --volume /etc/yum.repos.internal.d:/etc/yum.repos.internal.d --volume /etc/yum.conf:/etc/yum.conf -t container-registry.oracle.com/olcne/ceph-csi-operator:v0.4.0-1 -f ./olm/builds/Dockerfile .
Error: validating volumes: faccessat /etc/yum.repos.internal.d: no such file or directory
```